tf_yarn/examples/run_pytorch_examples.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Run this script from root dir ./examples/run_examples.sh
+#
+# Run all examples available with file name pattern *_example.py
+# This script prepares a virtual environment and example specific setup like downloading data.
+
+echo "run examples .."
+
+exit_code=0
+# sudo apt install liblzma-dev lzma
```

Comment:
remove if useless

Reply:
It is necessary, however I prefer not to perform sudo tasks in a script, so I left it as a comment. I'll rephrase